middleware/browse/browse_test.go

```diff
@@ -1,14 +1,16 @@
 package browse
 
 import (
+        "encoding/json"
+	"github.com/mholt/caddy/middleware"
```

Comment:
Oh ya , i removed these comments via the online editor on the page !! I have Gofmt integrated into my sublime editor !! My bad , I shall get it right in the next commit for the same test :)